### PR TITLE
Add Hugo Symbols to transcript endpoint

### LINKED ISSUE #128: Create Endpoint for getting ensembl id of all isoforms given a hugo symbol

### PR DESCRIPTION
**Pull request by inodb**
Fix #128

### CODE REVIEW

File: model/src/main/java/org/cbioportal/genome_nexus/model/EnsemblTranscript.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
     @Field(value="gene_stable_id")
     private String geneId;
 
+    @Field(value="hgnc_symbol")
```
Comment (onursumer) [inline model/src/main/java/org/cbioportal/genome_nexus/model/EnsemblTranscript.java]: maybe name the field as `hgnc_symbols`?

**Comment by inodb** [comment]
Split the data import part to https://github.com/genome-nexus/genome-nexus-importer/pull/1

File: docker-compose.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ services:
     depends_on:
       - db
   db:
-    image: genomenexus/gn-mongo:latest
+    build: https://github.com/inodb/genome-nexus-importer.git#add-exon-info-to-transcript
```
Comment (inodb) [inline docker-compose.yml]: points to `add-exon-info-to-transcript`, there's a fix in there to change `hgnc_symbol` to `hgnc_symbols` in the ensembl transcripts json used for import

File: service/src/main/java/org/cbioportal/genome_nexus/service/internal/EnsemblServiceImpl.java [hunk 3]
```diff
@@ -4,10 +4,12 @@
 import org.cbioportal.genome_nexus.persistence.EnsemblRepository;
 import org.cbioportal.genome_nexus.service.EnsemblService;
 import org.cbioportal.genome_nexus.service.exception.EnsemblTranscriptNotFoundException;
+import org.cbioportal.genome_nexus.service.exception.UnexpectedQueryParametersException;
```
Comment (onursumer) [inline service/src/main/java/org/cbioportal/genome_nexus/service/internal/EnsemblServiceImpl.java]: not used?

**Pull request merged**